### ISSUE TITLE
[Draft] Apps wire up the correct install, activate, manage, delete UI

### ISSUE THREAD

**Issue by ErisDS** [TryGhost/Ghost#2141]
This is a draft issue to go along with #2065 as it is anticipated that work will be required after #2083 and #2040 to get the app screens wired up exactly correctly.

**Comment by JohnONolan** [comment]
This is ancient - will be reopened with new issue/spec to reflect work that has happened in the last year
